tests/conftest.py

```diff
@@ -1394,7 +1399,7 @@ def tickers():
         'BLK/BTC': {
             'symbol': 'BLK/BTC',
             'timestamp': 1522014806072,
-            'datetime': '2018-03-25T21:53:26.720Z',
+            'datetime': '2018-03-25T21:53:26.072Z',
```

Comment:
that's for sure a typo in the small numbers.

bigger problem is that 2 tests are now failing.

Reply:
updated

Reply:
> that's for sure a typo in the small numbers.

Isn't 072 is the right value? Previously value was the typo.

Reply:
yeah you're right - if you instead of "updated" write "but this is the correct value now" i'd have had a closer look.

fixed in 60f2a12bd9104fb7233fafda6052102265f89013